src/Storage/Storage/Common/Cmdlet/NewAzureStorageContext.cs

```diff
@@ -178,6 +178,16 @@ public SwitchParameter Anonymous
 
         private bool isAnonymous;
 
+        [Parameter(HelpMessage = "Use OAuth storage account", Mandatory = false, ParameterSetName = SasTokenParameterSet)]
+        public SwitchParameter SASTokenWithConnectedAccount
+        {
+            get { return sasTokenWithConnectedAcount; }
+            set { sasTokenWithConnectedAcount = value; }
+        }
+
+        private bool sasTokenWithConnectedAcount = false;
+
+        [Parameter(HelpMessage = "Use OAuth storage account", ParameterSetName = SasTokenParameterSet)]
```

Comment:
"-EnableFileBackupRequestIntent" should also be added to SAS parameter set to support UD sas for File service.